x-pack/plugins/security_solution/public/management/cypress/tasks/create_endpoint_host.ts

```diff
@@ -17,6 +17,6 @@ export const createEndpointHost = (
     {
       agentPolicyId,
     },
-    { timeout: timeout ?? 600000 }
+    { timeout: timeout ?? 900000 } // 15 minutes, since setup can take 10 minutes and more. Task will time out if is not resolved within this time.
```

Comment:
I don't think so. If cypress task that set ups and enrols agent doesn't resolve within this timeout the task and test fails. We have noticed, that at some occurrences this 10 minutes is not enough to download, install, enroll and wait for agent to show up in fleet and test fails making it flaky not because of the test itself but this setup task. We could investigate a way to make this task faster or break it into several tasks but that should be a spike, this PR aims at stabilizing our test runs.